.vonage/catalog-info.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: backstage.io/v1alpha1
+kind: System
+metadata:
+  name: video-react-native-sdk
+  title: Video React Native SDK
```

Comment:
Note for the reviewer: Description for this is `Video React Natvie SDKs` in "Video - Cloud & Repo Ownerships" spreadsheet. I understand there is a typo there.